packages/tools/client-debugger/client-debugger-view/src/components/data-object-views/FluidObjectView.tsx

```diff
@@ -41,7 +41,7 @@ export function FluidObjectView(props: FluidObjectViewProps): React.ReactElement
 		fluidObjectHandle.get().then(setResolvedData, (error) => {
 			throw error;
 		});
-	}, [resolvedData]);
+	}, [fluidObjectHandle, setResolvedData]);
```

Comment:
I think er probably still need the `resolvedData` in list?

Reply:
In this case, the effect never actually observes the state of `resolvedData`, so it doesn't need to know when that changes. Since `resolvedData` is derived from the `fluidObjectHandle`, it's `fluidObjectHandle` that is needed here.

If we specify `resolvedData` here, then this effect will end up re-running every time it calls `setResolvedData`, which we don't want (and could cause bugs if we did anything unconditionally within the effect.